bin/s2nd.c

```diff
@@ -169,13 +169,13 @@ int cache_store_callback(struct s2n_connection *conn, void *ctx, uint64_t ttl, c
     S2N_ERROR_IF(key_size == 0 || key_size > MAX_KEY_LEN, S2N_ERR_INVALID_ARGUMENT);
     S2N_ERROR_IF(value_size == 0 || value_size > MAX_VAL_LEN, S2N_ERR_INVALID_ARGUMENT);
 
-    uint8_t index = ((const uint8_t *)key)[0];
+    uint8_t position = ((const uint8_t *)key)[0];
```

Comment:
A more usual direct name substitution would be "idx" or "indx".

In this specific case, "cache_index" or "cache_i" would also be more descriptive.